lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1082,6 +1082,7 @@ def cla(self):
         # Disabling mouse interaction might have been needed a long
         # time ago, but I can't find a reason for it now - BVR (2012-03)
         #self.disable_mouse_rotation()
+        Axes.cla(self)
```

Comment:
I've battled with `super` quite a bit. If there is a bug, it usually has to do with Method Resolution Order (MRO) (I hope...). I think you know that already, but writing it in case anyone else is reading.

But that's something you need to worry about only when a class inherits from multiple parents. Multiple inheritance complicates things, if you just try to think about how you would do it if you had to using only `super`, you quickly see it's not that simple.

Also, I'm not sure if MRO changed from python 2x to 3x. I'll do some reading and see if it's worth doing. So I'll give it a shot (and keep it simple), and hope your bugs were to do with MRO...